lib/config_cat/api.ex

```diff
@@ -0,0 +1,11 @@
+defmodule ConfigCat.API do
+  use HTTPoison.Base
+
+  def process_request_headers(headers) do
+    [
+      {"User-Agent", "ConfigCat-Elixir/m-0.0.1"},
```

Comment:
Depending on the polling mode this should be:
- manual poll: ConfigCat-Elixir/m-
- auto poll: ConfigCat-Elixir/a-
- lazy loading: ConfigCat-Elixir/l-

Also please fetch the version from the mix.exs if possible.

Reply:
Extracted to #16 and #17.

Good catch on the version! Since this was part of our app, we didn't have a separate mix.exs for it at the time.  And I forgot about going back to update those prefixes after we added more polling modes.